packages/api-file-manager/src/handlers/transform/optimizeImage.ts

```diff
@@ -3,28 +3,33 @@
  */
 // @ts-ignore
```

Comment:
Let's get rid of this ignore, and add `"@types/sharp": "^0.32.0"` to `devDependencies`. We'll get proper autocomplete then.